packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3949,6 +3954,7 @@ export class ContainerRuntime
 					pending,
 					pendingAttachmentBlobs,
 					pendingIdCompressorState,
+					sessionExpiryTimerStarted: this.garbageCollector.sessionExpiryTimerStarted,
 				};
```

Comment:
Consider adding a doc comment to `getPendingLocalState`. What is it used for? What's in the pending state?